File: c++/Ram.cpp
class Ram{
	private:
		/*atribut private*/
		string capacity;
		string price;

	public:
		/*constructor*/
		Ram(){}
		Ram(string capacity, string price){
			this->capacity = capacity;
			this->price = price;
		}

		/*getter/setter*/
		void setcapacity(string capacity){
			this->capacity = capacity;
		}
		string getcapacity(){
			return this->capacity;
		}
		void setprice(string price){
			this->price = price;
		}
		string getprice(){
			return this->price;
		}

		/*Method*/
		void printRam(){
			cout << "Ram capacity : " << this->capacity << endl;
			cout << "Ram price : " << this->price << endl;
		}
		/*destructor*/
		~Ram(){}
};

File: python/Ram.py
class Ram:
	# atribut private
	__capacity = ""
	__price = ""

	# constructor
	def __init__(self, capacity, price):
		self.__capacity = capacity
		self.__price = price

	# getter/setter
	def setcapacity(self, capacity):
		self.__capacity = capacity

	def getcapacity(self):
		return self.__capacity

	def setprice(self, price):
		self.__price = price

	def getprice(self):
		return self.__price

	# method
	def printRam(self):
		print("Ram capacity : " + str(self.__capacity))
		print("Ram price : " + str(self.__price))
